messenger/messenger.go

```diff
@@ -12,21 +12,32 @@ import (
 	"os"
 	"os/signal"
 	"strconv"
+	"sync"
 
 	log "github.com/sirupsen/logrus"
 
 	astriaGrpc "buf.build/gen/go/astria/execution-apis/grpc/go/astria/execution/v1alpha2/executionv1alpha2grpc"
 	"github.com/gorilla/mux"
+	"github.com/gorilla/websocket"
+	"github.com/rs/cors"
 	"google.golang.org/grpc"
 )
 
+var wsUpgrader = websocket.Upgrader{
+	ReadBufferSize:  1024,
+	WriteBufferSize: 1024,
+	CheckOrigin: func(r *http.Request) bool {
+		return true // Allow all connections
+	},
+}
+
 type Config struct {
 	SequencerRPC string `env:"SEQUENCER_RPC, default=http://localhost:26657"`
 	ConductorRPC string `env:"CONDUCTOR_RPC, default=:50051"`
 	RESTApiPort  string `env:"RESTAPI_PORT, default=:8080"`
 	RollupName   string `env:"ROLLUP_NAME, default=messenger-rollup"`
 	RollupID     string `env:"ROLLUP_ID, default=messenger-rollup"`
-	SeqPrivate   string `env:"SEQUENCER_PRIVATE, default="`
+	SeqPrivate   string `env:"SEQUENCER_PRIVATE, default=00fd4d6af5ac34d29d63a04ecf7da1ccfcbcdf7f7ed4042b8975e1c54e96d685"`
```

Comment:
nit: i would put the defaults in the .env instead of hardcoding into the go code. feels easier to read imo but maybe thats just me

Reply:
I agree, there was some issue at some where the env var was not working and I added it as a default to move forward. This is not the case now. I'll revert this.